MTM-64344 document removing attachment references fix (#3664)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-284-0-remove-child-references-from-parents-of-deleted-attachement.md

---

> date: '2025-08-26'

Reviewer: ```suggestion
date: 
```

title: Delete child references from parents of deleted attachment
change_type:
  - value: change-2c7RdTdXo4
    label: Improvement
product_area: Platform services
component:
  - value: component-JlFdtOPva
    label: REST API
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-64344
version: 2025.284.0
---

> Remove child references from parents of deleted attachment.

Reviewer: ```suggestion
```

> When an attachment is deleted, the references to this attachment are now also removed from the parent managed objects. This ensures that there are no leftover references pointing to no-existing managed objects.

Reviewer: ```suggestion
When an attachment is deleted, the references to this attachment are now also removed from the parent managed objects. This ensures that no leftover references are pointing to non-existent managed objects.
```
